Documentation/network/bgp-control-plane.rst

```diff
@@ -19,6 +19,21 @@ Currently a single flag in the ``Cilium Agent`` exists to turn on the
 
    --enable-bgp-control-plane=true
 
+Following values have to be set for helm installation
```

Comment:
OK, I'd maybe rephrase like this:

```suggestion
If using Helm charts instead, the relevant values are the following:
```

Nit: You also have a typo in the commit description for this change (`arument` -> `argument`).